### PR TITLE
Add monitoring data purge policy

### LINKED ISSUE #282: Purge old data?
How is old data purged from the temboard server database?

### PR DESCRIPTION
**Pull request by julmon**
Fixes #282

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -2,29 +2,141 @@
...
+  Twillio account SID.
+  Default: None
+
+  - **twilio_auth_token**
+  Twilio authentication token.
+  Default: None
+
+  - **twilio_from**
+  Twilio from phone number
+  Default: None
```
Comment (dlax) [inline docs/configuration.md]: I got distracted by the mix of formatting changes and addition of the "monitoring" section below. Would have been nice to have the former in a separate commit.

File: requirements-ci.txt [hunk 2]
```diff
@@ -5,4 +5,5 @@ pytest-mock
 pytest-tornado
 mock
 docutils
+psycopg2-binary
```
Comment (dlax) [inline requirements-ci.txt]: Why this change?

File: docs/configuration.md [hunk 3]
```diff
@@ -2,29 +2,131 @@
...
+
+  - **ssl_ca_cert_file**
+  File where to store each agent's SSL certificate. Comment it to disable SSL
+  certifcate checks.
+  Default: None
+
+  - **cookie_secret**
+  Secret key used to crypt cookie content.
+  Default: None;
```
Comment (dlax) [inline docs/configuration.md]: Default values have been changed for these options. Is this intended?

File: docs/configuration.md [hunk 4]
```diff
@@ -2,29 +2,131 @@
...
+
+
+### `repository`
+
+Connection parameters to the data repository aka `temboard` database.
+
+
+  - **host**
+  Repository host name or address.
+  Default: `/var/run/postgresql`
```
Comment (dlax) [inline docs/configuration.md]: Same here, the default value has changed.

File: docs/configuration.md [hunk 5]
```diff
@@ -2,29 +2,131 @@
...
+  Repository port number.
+  Default: `5432`
+
+  - **user**
+  Connection user.
+  Default: `temboard`
+
+  - **password**
+  User password.
+  Default: `temboard`
```
Comment (dlax) [inline docs/configuration.md]: Also changed

File: docs/configuration.md [hunk 6]
```diff
@@ -2,29 +2,131 @@
...
+
+
+### `logging`
+
+How and where to log.
+
+
+  - **method**
+  Method used to send the logs: `stderr`, `syslog` or `file`.
+  Default: `stderr`
```
Comment (dlax) [inline docs/configuration.md]: Changed

File: docs/configuration.md [hunk 7]
```diff
@@ -2,29 +2,131 @@
...
+  Twillio account SID.
+  Default: None
+
+  - **twilio_auth_token**
+  Twilio authentication token.
+  Default: None
+
+  - **twilio_from**
+  Twilio from phone number
+  Default: None
```
Comment (dlax) [inline docs/configuration.md]: This section is new?

Comment (pgiraud) [reply docs/configuration.md]: It's not new. I forgot to document it when I worked on the alert notifications feature. My bad.

File: requirements-ci.txt (hunk 2 above)
Comment (julmon) [reply requirements-ci.txt]: Because of this https://github.com/julmon/temboard/blob/purge_data/temboardui/plugins/monitoring/__init__.py#L14

File: docs/configuration.md (hunk 3 above)
Comment (julmon) [reply docs/configuration.md]: Actually, previous documentation wasn't right, the default parameters values have been taken from `temboard.conf` template file shipped in `share/` dir., but those values are not the real default values.

File: docs/configuration.md (hunk 4 above)
Comment (julmon) [reply docs/configuration.md]: Please see https://github.com/dalibo/temboard/pull/689#discussion_r387899567

File: docs/configuration.md (hunk 5 above)
Comment (julmon) [reply docs/configuration.md]: https://github.com/dalibo/temboard/pull/689#discussion_r387899567

File: docs/configuration.md (hunk 6 above)
Comment (julmon) [reply docs/configuration.md]: https://github.com/dalibo/temboard/pull/689#discussion_r387899567

File: docs/configuration.md (hunk 7 above)
Comment (julmon) [reply docs/configuration.md]: Those changes are now applied from their own commit.

File: docs/configuration.md (hunk 3 above)
Comment (dlax) [reply docs/configuration.md]: Ok, I see.
